frontend/components/hourInput.spec.tsx

```diff
@@ -1,18 +1,23 @@
-import { fireEvent, getByDisplayValue, render, screen } from '@testing-library/react'
+import React from 'react'
+import { fireEvent, render, screen } from '@testing-library/react'
 import { HourInput } from './hourInput'
-import userEvent from '@testing-library/user-event'
 
 describe('the hour input control should display ...', () => {
     const testNode = (
         <>
-            <HourInput></HourInput>
+            <HourInput
+                workHours={1}
+                onChange={(workHours): void => {
+                    console.log(workHours)
+                }}
+            ></HourInput>
             <button>Click me!</button>
         </>
     )
 
-    it('... 0:00 in the beginning', async () => {
+    it('... 0:00 in the beginning', () => {
         const { findByDisplayValue } = render(testNode)
-        expect(await findByDisplayValue('0:00')).toBeDefined()
```

Comment:
seems wrong to me because findBy... is always defined because it is a promise, not the result of a promise....